starknet-crypto/Cargo.toml

```diff
@@ -24,7 +24,7 @@ rfc6979 = { version = "0.4.0", default-features = false }
 sha2 = { version = "0.10.6", default-features = false }
 zeroize = { version = "1.6.0", default-features = false }
 hex = { version = "0.4.3", default-features = false, optional = true }
-starknet-types-core = { version = "0.1.7", default-features = false, features = ["curve", "hash"] }
+starknet-types-core = { version = "0.1.8", default-features = false, features = ["curve", "hash"] }
```

Comment:
Latest version of `starknet-types-core` support operations on EC points as well https://github.com/starknet-io/types-rs/commit/e0eff28053b0280629c3bf6827e6246995a27503